Week1/homework/app.js

```diff
@@ -1,11 +1,51 @@
 'use strict';
 {
   const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets'
+    { 'title': 'Becoming', 'language': 'English', 'author': 'Michelle Obama' },
+    { 'title': 'Those Who Knew', 'language': 'English', 'author': 'Scott Belsky' },
+    { 'title': 'Good and Mad', 'language': 'English', 'author': 'Rebecca Traister' },
+    { 'title': 'The Messy Middle', 'language': 'English', 'author': 'Scott Belsky' },
+    { 'title': 'Lake Success', 'language': 'English', 'author': 'Gary Shteyngart' },
+    { 'title': 'Fashion Climbing', 'language': 'English', 'author': 'Bill Cunningham' },
+    { 'title': 'Crudo', 'language': 'English', 'author': 'Olivia Laing' },
+    { 'title': 'Winners Take All', 'language': 'English', 'author': 'Anand Giridharadas' },
+    { 'title': 'French Exit', 'language': 'English', 'author': 'Patrick deWitt' },
+    { 'title': 'How To Be Famous', 'language': 'English', 'author': 'Caitlin Moran' }
   ];
 
+  function makeList() {
```

Comment:
This is purely a personal call, but I would have broken all the functionality in this function into a few different functions. (Actually, I did do that when I did the homework. 😄 ).
It allows you to test each part of the layout code separately.